### DISCUSSION TITLE
How to use the docker version DynAdjust

### DISCUSSION THREAD

**Discussion by yohnnylim** [GeoscienceAustralia/DynAdjust#211] [Q&A] [answered]
Hi,

I've come across the DynAdjust in Docker hub and am wondering how do I run the Least Squares processing using docker?
Any help would be highly appreciated.

**Comment by rogerfraser** [comment, marked as answer]
Hi @yohnnylim, 

I'll assume you're familiar with Docker and have it installed on your machine.  If not, I suggest familiarising yourself with the basics on https://docs.docker.com/.

To run the DynAdjust docker image locally, install the DynAdjust docker image (https://hub.docker.com/r/icsm/dynadjust) via the pull command.  For example:
  `$> docker pull icsm/dynadjust`

Once the image has been installed, create the container (via the `docker run` command) and then execute the DynAdjust binaries from `/opt/dynadjust` using the `docker exec` command on the container.  See the [exec help](https://docs.docker.com/engine/reference/commandline/exec/) for more information.

Hope this helps.

**Comment by yohnnylim** [comment]
Thanks alot! this answers the question
